ckanext/dcatapit/dcat/profiles.py

```diff
@@ -1025,10 +1032,20 @@ def _add_agent(self, _dict, ref, basekey, _type, use_default_lang=False, agent_d
         self.g.add((agent, RDF['type'], FOAF.Agent))
         self.g.add((ref, _type, agent))
 
+        def _found_no_lang():
+            found_no_lang = False
+            for val in self.g.objects(agent, FOAF.name):
+                 if not hasattr(val, 'lang') or not val.lang:
+                     found_no_lang = True
+                     break
+            return found_no_lang
+
         if isinstance(agent_name, dict):
             for lang, aname in agent_name.items():
                 if lang in OFFERED_LANGS:
                     self.g.add((agent, FOAF.name, Literal(aname, lang=lang_mapping_ckan_to_xmllang.get(lang, lang))))
+                    if lang == DEFAULT_LANG and not _found_no_lang():
+                        self.g.add((agent, FOAF.name, Literal(aname)))
```

Comment:
What if a FOAF:name without lang is found? It seems that it will not added to the graph.

Reply:
Thx for alert. Here we create FOAF:name entries. If agent_name is dictionary, that means it's multilang value (with keys as lang codes). There's else case beneath, which handles plain string value (no lang case):
https://github.com/cezio/ckanext-dcatapit/blob/c956ca2fbd2f6e7acb403e23be23e5c956aa4266/ckanext/dcatapit/dcat/profiles.py#L1043-L1053